app/src/shared/hooks/usePending.jsx

```diff
@@ -35,8 +76,8 @@ export function usePending(name: string) {
 }
 
 export function useAnyPending() {
-    const { pending } = useContext(PendingContext)
-    return Object.values(pending).some((isPending) => isPending)
+    const { isPending } = useContext(PendingContext)
```

Comment:
`useAnyPending` doesn't seem to be covered by any test.

Reply:
Fixed in e405bcf